src/extension.ts

```diff
@@ -368,7 +368,7 @@ export class Extension implements RunHooks {
       if (error) {
         if (error.location) {
           const document = await this._vscode.workspace.openTextDocument(error.location.file);
-          const position = new this._vscode.Position(error.location.line - 1, error.location.column - 1);
+          const position = new this._vscode.Position(Math.max(0, error.location.line - 1), error.location.column - 1);
```

Comment:
These changes seem unrelated.

Reply:
They're unrelated, but I stumbled into weird errors here while working on this change. The `Position` constructor threw because of `-1` being passed. Let me try reproing them.